src/ar/ar.json

```diff
@@ -1,5 +1,5 @@
 /**
- * This file was automatically generated and contains unvalidated content
+ * This file contains unvalidated content
  * Once verified, add the base IETF language tag to the array in the
  * src/validated-languages.json file and remove this comment
  */
```

Comment:
Oh, I meant you could remove the entire comment, since you're validating the content. Leaving it as it is now makes it sound like no one has looked and checked any of the data in the file.